arwen/contexts/blockchain.go

```diff
@@ -108,18 +117,24 @@ func (context *blockchainContext) GetCodeHash(addr []byte) ([]byte, error) {
 	return context.host.Crypto().Keccak256(code)
 }
 
-func (context *blockchainContext) GetCode(addr []byte) ([]byte, error) {
+func (context *blockchainContext) GetCode(address []byte) ([]byte, error) {
 	// TODO must get the code from the OutputAccount, if present
```

Comment:
delete toDO

Reply:
Deleted, and added Jira task: https://elrondnetwork.atlassian.net/browse/EN-6829